milestone1

```diff
@@ -1 +1,69 @@
 
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ToDoList<T> {
+    private ArrayList<T> tasks;
+
+    public ToDoList() {
+        tasks = new ArrayList<>();
+    }
+
+    public void addTask(T task) {
+        tasks.add(task);
+        System.out.println("Task added: " + task);
+    }
+
+    public void displayTasks() {
+        if (tasks.isEmpty()) {
+            System.out.println("No tasks in the list.");
+        } else {
+            System.out.println("Your To-Do List:");
+            for (int i = 0; i < tasks.size(); i++) {
+                System.out.println((i + 1) + ". " + tasks.get(i));
+            }
+        }
+    }
+
+    public void removeTask(int index) {
```

Comment:
Since you allow task removal consider using a LinkedList instead of an ArrayList. Use an iterator instead of get to print the list of tasks.